src/plugins/intel_cpu/src/nodes/concat.cpp

```diff
@@ -22,7 +22,7 @@
 #include "common/cpu_memcpy.h"
 #include "common/blocked_desc_creator.h"
 #include <memory_desc/cpu_memory_desc_utils.h>
-
+#include "common/dnnl_thread.hpp"
```

Comment:
It might be better to use OV parallel routines, please see `src/inference/include/ie/ie_parallel.hpp`, just to have less dependency on oneDNN.

Reply:
The `parallel_for` in ov supports 5d at most, but we need `parallel_for6d`, should we implement `parallel_for6d` ?

Reply:
I think yes, why not, if it is not too complicated.

Reply:
Thanks for advice, `parallel_for6d`  implementation in 30ac864a3f5a79bb8bca6b39bac14172a21745b4